MechoSoma/XTool/port.cpp

```diff
@@ -102,7 +102,9 @@ char* port_ultoa( unsigned long value, char* result, int base ) {
 #ifndef _WIN32
 char *_fullpath(char *absPath, const char *relPath, size_t maxLength)
 {
-	return realpath(relPath, absPath);
```

Comment:
`realpath` function can return NULL if the last component is not exists, so we need another API to get full path

Reply:
can we delete relpath function now?